modules/planning/reference_line/reference_line_provider.cc

```diff
@@ -480,10 +480,14 @@ bool ReferenceLineProvider::GetNearestWayPointFromNavigationPath(
     double s = 0.0;
     double l = 0.0;
     if (!lane->GetProjection({point.x(), point.y()}, &s, &l)) {
+      AERROR << "get adc projection point on lane failed";
       continue;
     }
     constexpr double kEpsilon = 1e-6;
-    if (s > (lane->total_length() + kEpsilon) || (s + kEpsilon) < 0.0) {
+    if (s > (lane->total_length() + kEpsilon)) {
```

Comment:
Why removed the second condition?

Reply:
Under the relative map, the initial s does not always start from 0.0, and many times the s of the first few points is negative.
if do't remove ,have maney such error:
reference_line_provider.cc:537] failed to find nearest point x: 0 y: 0

Reply:
Is it possible to treat this issue inside navigation mode? Other modules could be affected if there are changes on shared API.

Reply:
There is no suitable local revision at present,
i will revoke this amendment.